src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -118,4 +122,42 @@ public OpenLoadFlowParameters setLowImpedanceBranchMode(LowImpedanceBranchMode l
     public List<AcLoadFlowObserver> getAdditionalObservers() {
         return additionalObservers;
     }
+
+    @AutoService(LoadFlowParameters.ConfigLoader.class)
+    public static class OpenLoadFlowConfigLoader implements LoadFlowParameters.ConfigLoader<OpenLoadFlowParameters> {
+
+        @Override
+        public OpenLoadFlowParameters load(PlatformConfig platformConfig) {
+            OpenLoadFlowParameters parameters = new OpenLoadFlowParameters();
+
+            platformConfig.getOptionalModuleConfig("open-loadflow-default-parameters")
+                    .ifPresent(config -> {
+                        parameters.setBalanceType(config.getEnumProperty(BALANCE_TYPE_PARAM_NAME, BalanceType.class, BALANCE_TYPE_DEFAULT_VALUE));
+                        parameters.setDc(config.getBooleanProperty(DC_PARAM_NAME, DC_DEFAULT_VALUE));
+                        parameters.setDistributedSlack(config.getBooleanProperty(DISTRIBUTED_SLACK_PARAM_NAME, DISTRIBUTED_SLACK_DEFAULT_VALUE));
+                        parameters.setLowImpedanceBranchMode(config.getEnumProperty(LOW_IMPEDANCE_BRANCH_MODE_PARAM_NAME, LowImpedanceBranchMode.class, LOW_IMPEDANCE_BRANCH_MODE_DEFAULT_VALUE));
+                        parameters.setVoltageRemoteControl(config.getBooleanProperty(VOLTAGE_REMOTE_CONTROLE_PARAM_NAME, VOLTAGE_REMOTE_CONTROLE_DEFAULT_VALUE));
+                        parameters.setThrowsExceptionInCaseOfSlackDistributionFailure(
+                                config.getBooleanProperty(THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_PARAM_NAME, THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_DEFAULT_VALUE)
+                        );
```

Comment:
I'm wondering why thoses lines are considered uncovered by the test, any ideas what we're missing?

Reply:
It might be possible that sonar could not trace the cal that go througth configLoader in an other package